src/app/containers/VestedHistory/components/VestedPagination.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+import { Button, ButtonStyle } from 'app/components/Button';
+
+interface IVestedPaginationProps {
+  page: number;
+  loading: boolean;
+  isDisabled: boolean;
+  onChange: (value: number) => void;
+}
+
+export const VestedPagination: React.FC<IVestedPaginationProps> = ({
```

Comment:
I think that the exact same pagination is used for every graph migration story as we don't support proper pagination. I know you cannot reuse it now because neither of the stories is merged but pls create a Monday.com story (in backlog) for creating universal pagination once some of the stories are merged.

Reply:
https://sovryn.monday.com/boards/2218344956/pulses/2968859815